lighthouse-core/gather/gather-runner.js

```diff
@@ -491,14 +501,21 @@ class GatherRunner {
 
   static instantiateGatherers(passes, rootPath) {
     return passes.map(pass => {
-      pass.gatherers = pass.gatherers.map(gatherer => {
+      pass.gatherers = pass.gatherers.map(gathererDefn => {
         // If this is already instantiated, don't do anything else.
-        if (typeof gatherer !== 'string') {
-          return gatherer;
+        if (gathererDefn.instance) {
+          return gathererDefn;
+        }
+
+        if (typeof gathererDefn.beforePass === 'function') {
```

Comment:
this is for backwards compat? if so can you add a comment with the case this covers?

Reply:
done